### PR TITLE
Assorted language changes

### PR DESCRIPTION
**Pull request by bentito**

### CODE REVIEW

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java [hunk 1]
```diff
@@ -341,27 +342,29 @@ public CassandraClientConfig setTokenAwareShuffleReplicas(boolean tokenAwareShuf
...
-    public boolean isUseWhiteList()
+    public boolean isUseAllowList()
     {
-        return this.useWhiteList;
+        return this.useAllowList;
     }
 
-    @Config("cassandra.load-policy.use-white-list")
-    public CassandraClientConfig setUseWhiteList(boolean useWhiteList)
+    @Config("cassandra.load-policy.use-allow-list")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:350]: 👍

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientModule.java [hunk 2]
```diff
@@ -122,13 +122,14 @@ public static CassandraSession createCassandraSession(CassandraClientConfig conf
             loadPolicy = new TokenAwarePolicy(loadPolicy, config.isTokenAwareShuffleReplicas());
         }
 
-        if (config.isUseWhiteList()) {
-            checkArgument(!config.getWhiteListAddresses().isEmpty(), "empty WhiteListAddresses");
-            List<InetSocketAddress> whiteList = new ArrayList<>();
-            for (String point : config.getWhiteListAddresses()) {
-                whiteList.add(new InetSocketAddress(point, config.getNativeProtocolPort()));
+        if (config.isUseAllowList()) {
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientModule.java:125]: 👍

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java [hunk 3]
```diff
@@ -341,27 +342,29 @@ public CassandraClientConfig setTokenAwareShuffleReplicas(boolean tokenAwareShuf
...
-    public List<String> getWhiteListAddresses()
+    public List<String> getAllowListAddresses()
     {
-        return whiteListAddresses;
+        return allowListAddresses;
     }
 
-    @Config("cassandra.load-policy.white-list.addresses")
-    public CassandraClientConfig setWhiteListAddresses(String commaSeparatedList)
+    @Config("cassandra.load-policy.allow-list.addresses")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:363]: 👍

File: presto-spi/src/main/java/io/prestosql/spi/predicate/DiscreteValues.java [hunk 4]
```diff
@@ -20,7 +20,7 @@
     /**
      * @return true if the values are to be included, false if the values are to be excluded
      */
-    boolean isWhiteList();
+    boolean isAllowList();
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/main/java/io/prestosql/spi/predicate/DiscreteValues.java:23]: 👍

File: presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java [hunk 5]
```diff
@@ -42,20 +42,20 @@
...
- * or collectively excluded (aka !whitelist).
+ * Assumes an infinite number of possible values. The values may be collectively included (aka allowList)
+ * or collectively excluded (aka !allowList).
  */
 public class EquatableValueSet
         implements ValueSet
 {
     private final Type type;
-    private final boolean whiteList;
+    private final boolean allowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java:52]: 👍

File: presto-thrift-api/src/main/java/io/prestosql/plugin/thrift/api/valuesets/PrestoThriftEquatableValueSet.java [hunk 6]
```diff
@@ -33,27 +33,27 @@
...
- * or collectively excluded (aka !whitelist).
+ * Assumes an infinite number of possible values. The values may be collectively included (aka allowList)
+ * or collectively excluded (aka !allowList).
  * This structure is used with comparable, but not orderable types like "json", "map".
  */
 @ThriftStruct
 public final class PrestoThriftEquatableValueSet
 {
-    private final boolean whiteList;
+    private final boolean allowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-thrift-api/src/main/java/io/prestosql/plugin/thrift/api/valuesets/PrestoThriftEquatableValueSet.java:43]: 👍

File: presto-docs/src/main/sphinx/connector/cassandra.rst [hunk 7]
```diff
@@ -137,9 +137,9 @@ Property Name                                                 Description
 ``cassandra.load-policy.shuffle-replicas``                    Set to ``true`` to use ``TokenAwarePolicy`` with shuffling of replicas,
                                                               defaults to ``false``.
 
-``cassandra.load-policy.use-white-list``                      Set to ``true`` to use ``WhiteListPolicy``, defaults to ``false``.
+``cassandra.load-policy.use-allow-list``                      Set to ``true`` to use ``AllowListPolicy``, defaults to ``false``.
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-docs/src/main/sphinx/connector/cassandra.rst:140]: ✅

File: presto-docs/src/main/sphinx/connector/cassandra.rst [hunk 8]
```diff
@@ -137,9 +137,9 @@ Property Name                                                 Description
...
 
-``cassandra.load-policy.white-list.addresses``                Comma-separated list of hosts for ``WhiteListPolicy``.
+``cassandra.load-policy.allow-list.addresses``                Comma-separated list of hosts for ``AllowListPolicy``.
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-docs/src/main/sphinx/connector/cassandra.rst:142]: ✅

File: presto-docs/src/main/sphinx/include/PrestoThriftService.thrift [hunk 9]
```diff
@@ -58,12 +58,12 @@ struct PrestoThriftAllOrNoneValueSet {
...
  * A set containing values that are uniquely identifiable.
- * Assumes an infinite number of possible values. The values may be collectively included (aka whitelist)
- * or collectively excluded (aka !whitelist).
+ * Assumes an infinite number of possible values. The values may be collectively included (aka allowList)
+ * or collectively excluded (aka !allowList).
  * This structure is used with comparable, but not orderable types like "json", "map".
  */
 struct PrestoThriftEquatableValueSet {
-  1: bool whiteList;
+  1: bool allowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-docs/src/main/sphinx/include/PrestoThriftService.thrift:66]: ✅

File: presto-docs/src/main/sphinx/release/release-0.153.rst [hunk 10]
```diff
@@ -151,7 +151,7 @@ SPI Changes
 * Make ``NodeManager`` specific to a connector instance.
 * Replace ``ConnectorFactoryContext`` with ``ConnectorContext``.
 * Use ``@SqlNullable`` for functions instead of ``@Nullable``.
-* Use a whitelist model for plugin class loading. This prevents connectors
+* Use an allow-list model for plugin class loading. This prevents connectors
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-docs/src/main/sphinx/release/release-0.153.rst:154]: ✅

File: presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java [hunk 11]
```diff
@@ -1802,7 +1802,7 @@ else if (firstException != exception) {
         throw propagate(firstException);
     }
 
-    // TODO instead of whitelisting exceptions we propagate we should recognize exceptions which we suppress and try different alternative call
+    // TODO instead of allow-listing exceptions we propagate we should recognize exceptions which we suppress and try different alternative call
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java:1805]: ✅

File: presto-hive/src/main/java/io/prestosql/plugin/hive/util/RetryDriver.java [hunk 12]
```diff
@@ -41,7 +41,7 @@
     private final Duration maxSleepTime;
     private final double scaleFactor;
     private final Duration maxRetryTime;
-    private final List<Class<? extends Exception>> exceptionWhiteList;
+    private final List<Class<? extends Exception>> exceptionAllowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/util/RetryDriver.java:44]: ✅

File: presto-main/src/main/java/io/prestosql/metadata/DiscoveryNodeManager.java [hunk 13]
```diff
@@ -198,8 +198,8 @@ public void refreshNodes()
 
     private synchronized void refreshNodesInternal()
     {
-        // This is currently a blacklist.
-        // TODO: make it a whitelist (a failure-detecting service selector) and maybe build in support for injecting this in airlift
+        // This is currently a deny-list.
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-main/src/main/java/io/prestosql/metadata/DiscoveryNodeManager.java:201]: ✅

File: presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java [hunk 14]
```diff
@@ -73,24 +73,24 @@ public void testSingleValue()
 
         EquatableValueSet complement = (EquatableValueSet) EquatableValueSet.all(ID).subtract(equatables);
 
-        // Whitelist
+        // allow-list
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java:76]: ✅

File: presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java [hunk 15]
```diff
@@ -73,24 +73,24 @@ public void testSingleValue()
...
-        assertTrue(equatables.isWhiteList());
+        assertTrue(equatables.isAllowList());
         assertTrue(Iterables.elementsEqual(equatables.getValues(), ImmutableList.of(10L)));
         assertEquals(equatables.complement(), complement);
         assertFalse(equatables.containsValue(0L));
         assertFalse(equatables.containsValue(1L));
         assertTrue(equatables.containsValue(10L));
 
-        // Blacklist
+        // deny-list
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java:88]: ✅

File: presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java [hunk 16]
```diff
@@ -105,12 +105,12 @@ public void testMultipleValues()
 
         EquatableValueSet complement = (EquatableValueSet) EquatableValueSet.all(ID).subtract(equatables);
 
-        // Whitelist
+        // allow-list
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java:108]: ✅

File: presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java [hunk 17]
```diff
@@ -119,12 +119,12 @@ public void testMultipleValues()
         assertTrue(equatables.containsValue(3L));
         assertFalse(equatables.containsValue(4L));
 
-        // Blacklist
+        // deny-list
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java:122]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Validator.java [hunk 18]
```diff
@@ -214,7 +214,7 @@ private boolean validate()
         try {
             controlResult = executePreAndMainForControl();
 
-            // query has too many rows. Consider blacklisting.
+            // query has too many rows. Consider using a deny-list.
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Validator.java:217]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 19]
```diff
@@ -55,17 +55,17 @@
     private final VerifierConfig config;
     private final Set<EventClient> eventClients;
     private final int threadCount;
-    private final Set<String> whitelist;
-    private final Set<String> blacklist;
+    private final Set<String> allowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:58]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 20]
```diff
@@ -55,17 +55,17 @@
...
+    private final Set<String> denyList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:59]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 21]
```diff
@@ -80,19 +80,19 @@ public int run(List<QueryPair> queries)
         int totalQueries = queries.size() * config.getSuiteRepetitions() * config.getQueryRepetitions();
         log.info("Total Queries:     %d", totalQueries);
 
-        log.info("Whitelisted Queries: %s", Joiner.on(',').join(whitelist));
+        log.info("Allow-listed Queries: %s", Joiner.on(',').join(allowList));
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:83]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 22]
```diff
@@ -80,19 +80,19 @@ public int run(List<QueryPair> queries)
...
 
         int queriesSubmitted = 0;
         for (int i = 0; i < config.getSuiteRepetitions(); i++) {
             for (QueryPair query : queries) {
                 for (int j = 0; j < config.getQueryRepetitions(); j++) {
-                    // If a whitelist exists, only run the tests on the whitelist
-                    if (!whitelist.isEmpty() && !whitelist.contains(query.getName())) {
-                        log.debug("Query %s is not whitelisted", query.getName());
+                    // If a allowList exists, only run the tests on the allowList
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:89]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 23]
```diff
@@ -80,19 +80,19 @@ public int run(List<QueryPair> queries)
...
+                    if (!allowList.isEmpty() && !allowList.contains(query.getName())) {
+                        log.debug("Query %s is not allow-listed", query.getName());
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:91]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java [hunk 24]
```diff
@@ -80,19 +80,19 @@ public int run(List<QueryPair> queries)
...
                         continue;
                     }
-                    if (blacklist.contains(query.getName())) {
-                        log.debug("Query %s is blacklisted", query.getName());
+                    if (denyList.contains(query.getName())) {
+                        log.debug("Query %s is deny-listed", query.getName());
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/Verifier.java:95]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 25]
```diff
@@ -58,8 +58,8 @@
     private String testGateway;
     private Duration controlTimeout = new Duration(10, TimeUnit.MINUTES);
     private Duration testTimeout = new Duration(1, TimeUnit.HOURS);
-    private Set<String> blacklist = ImmutableSet.of();
-    private Set<String> whitelist = ImmutableSet.of();
+    private Set<String> denyList = ImmutableSet.of();
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:61]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 26]
```diff
@@ -58,8 +58,8 @@
...
+    private Set<String> allowList = ImmutableSet.of();
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:62]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 27]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
+    public Set<String> getDenyList()
     {
-        return blacklist;
+        return denyList;
     }
 
-    @ConfigDescription("Names of queries which are blacklisted")
-    @Config("blacklist")
-    public VerifierConfig setBlacklist(String blacklist)
+    @ConfigDescription("Names of queries which are on the denyList")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:279]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 28]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
+    @Config("denyList")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:280]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 29]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
+    @LegacyConfig("blacklist")
+    public VerifierConfig setDenyList(String denyList)
     {
-        ImmutableSet.Builder<String> blacklistBuilder = ImmutableSet.builder();
-        for (String value : Splitter.on(',').trimResults().omitEmptyStrings().split(blacklist)) {
-            blacklistBuilder.add(value);
+        ImmutableSet.Builder<String> denylistBuilder = ImmutableSet.builder();
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:284]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 30]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
+    public Set<String> getAllowList()
     {
-        return whitelist;
+        return allowList;
     }
 
-    @ConfigDescription("Names of queries which are whitelisted. Whitelist is applied before the blacklist")
-    @Config("whitelist")
-    public VerifierConfig setWhitelist(String whitelist)
+    @ConfigDescription("Names of queries which are on the allowList. allowList is applied before the denyList")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:299]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 31]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
+    @Config("allowList")
+    @LegacyConfig("whitelist")
+    public VerifierConfig setAllowList(String allowList)
     {
-        ImmutableSet.Builder<String> whitelistBuilder = ImmutableSet.builder();
-        for (String value : Splitter.on(',').trimResults().omitEmptyStrings().split(whitelist)) {
-            whitelistBuilder.add(value);
+        ImmutableSet.Builder<String> allowlistBuilder = ImmutableSet.builder();
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:304]: ✅

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientModule.java [hunk 32]
```diff
@@ -122,13 +122,14 @@ public static CassandraSession createCassandraSession(CassandraClientConfig conf
...
+            checkArgument(!config.getAllowListAddresses().isEmpty(), "empty AllowListAddresses");
+            List<InetSocketAddress> allowList = new ArrayList<>();
+            for (String point : config.getAllowListAddresses()) {
+                allowList.add(new InetSocketAddress(point, config.getNativeProtocolPort()));
             }
-            loadPolicy = new WhiteListPolicy(loadPolicy, whiteList);
+            //TODO update this when com.datastax.driver is updated with regards to language
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientModule.java:131]: ✅

File: presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java [hunk 33]
```diff
@@ -42,20 +42,20 @@
...
- * or collectively excluded (aka !whitelist).
+ * Assumes an infinite number of possible values. The values may be collectively included
+ * or collectively excluded.
  */
 public class EquatableValueSet
         implements ValueSet
 {
     private final Type type;
-    private final boolean whiteList;
+    private final boolean allowList;
```
Comment (electrum) [inline presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java:52]: Please rename to `inclusive`

File: presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java [hunk 34]
```diff
@@ -42,20 +42,20 @@
...
     private final Set<ValueEntry> entries;
 
     @JsonCreator
     public EquatableValueSet(
             @JsonProperty("type") Type type,
-            @JsonProperty("whiteList") boolean whiteList,
+            @JsonProperty("allowList") boolean inclusive,
```
Comment (electrum) [inline presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java:58]: Change JSON property to `inclusive`

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 35]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
     {
-        return whitelist;
+        return allowedQueries;
     }
 
-    @ConfigDescription("Names of queries which are whitelisted. Whitelist is applied before the blacklist")
-    @Config("whitelist")
-    public VerifierConfig setWhitelist(String whitelist)
+    @ConfigDescription("Names of queries which are allowed. If non-empty, only allowed queries are used.")
+    @Config("allowList")
```
Comment (electrum) [inline presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:300]: Let's call this `allowed-queries`

Make sure to update `TestVerifierConfig` to match

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java [hunk 36]
```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
...
         }
 
-        this.blacklist = blacklistBuilder.build();
+        this.bannedQueries = bannedBuilder.build();
         return this;
     }
 
     @NotNull
-    public Set<String> getWhitelist()
+    public Set<String> getAllowQueries()
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:294]: ✅

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java [hunk 37]
```diff
@@ -67,8 +68,8 @@
     private boolean dcAwareAllowRemoteDCsForLocal;
     private boolean useTokenAware;
     private boolean tokenAwareShuffleReplicas;
-    private boolean useWhiteList;
-    private List<String> whiteListAddresses = ImmutableList.of();
+    private boolean useAllowList;
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:71]: ✅

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java [hunk 38]
```diff
@@ -341,27 +342,28 @@ public CassandraClientConfig setTokenAwareShuffleReplicas(boolean tokenAwareShuf
...
+    public List<String> getAllowedAddresses()
     {
-        return whiteListAddresses;
+        return allowedAddresses;
     }
 
-    @Config("cassandra.load-policy.white-list.addresses")
-    public CassandraClientConfig setWhiteListAddresses(String commaSeparatedList)
+    @Config("cassandra.load-policy.allowed.addresses")
+    @LegacyConfig("cassandra.load-policy.white-list.addresses")
```
[the remark this answers is not in the document]
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:363]: ✅

File: presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java (hunk 33 above)
Comment (bentito) [reply presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java:52]: ✅

File: presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java (hunk 34 above)
Comment (bentito) [reply presto-spi/src/main/java/io/prestosql/spi/predicate/EquatableValueSet.java:58]: ✅

File: presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java (hunk 35 above)
Comment (bentito) [reply presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java:300]: ✅

**Comment by bentito** [comment]
@electrum could you please suggest a fix for the Cassandra test failure? I think I took out the appropriate properties but now it's complaining about their absence. Grepping the code base didn't reveal where it was finding the properties to even generate the error for me.

File: presto-docs/src/main/sphinx/connector/cassandra.rst [hunk 39]
```diff
@@ -137,9 +137,7 @@ Property Name                                                 Description
 ``cassandra.load-policy.shuffle-replicas``                    Set to ``true`` to use ``TokenAwarePolicy`` with shuffling of replicas,
                                                               defaults to ``false``.
 
-``cassandra.load-policy.use-white-list``                      Set to ``true`` to use ``WhiteListPolicy``, defaults to ``false``.
-
-``cassandra.load-policy.white-list.addresses``                Comma-separated list of hosts for ``WhiteListPolicy``.
+``cassandra.load-policy.allow-list.addresses``                Comma-separated list of hosts to allow.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/connector/cassandra.rst:140]: Please change to `cassandra.load-policy.allowed-addresses`

File: presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java [hunk 40]
```diff
@@ -58,8 +58,6 @@ public void testDefaults()
                 .setDcAwareAllowRemoteDCsForLocal(false)
                 .setUseTokenAware(false)
                 .setTokenAwareShuffleReplicas(false)
-                .setUseWhiteList(false)
```
Comment (electrum) [inline presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java:61]: This needs to call `setAllowedAddresses()`

File: presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java [hunk 41]
```diff
@@ -99,8 +97,6 @@ public void testExplicitPropertyMappings()
                 .put("cassandra.load-policy.dc-aware.allow-remote-dc-for-local", "true")
                 .put("cassandra.load-policy.use-token-aware", "true")
                 .put("cassandra.load-policy.token-aware.shuffle-replicas", "true")
-                .put("cassandra.load-policy.use-white-list", "true")
-                .put("cassandra.load-policy.white-list.addresses", "host1")
```
Comment (electrum) [inline presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java:103]: Need to add
```java
.put("cassandra.load-policy.allowed-addresses", "host1,host2")

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java [hunk 42]
```diff
@@ -341,28 +340,9 @@ public CassandraClientConfig setTokenAwareShuffleReplicas(boolean tokenAwareShuf
...
-        return this;
-    }
-
-    public List<String> getWhiteListAddresses()
-    {
-        return whiteListAddresses;
-    }
-
-    @Config("cassandra.load-policy.white-list.addresses")
-    public CassandraClientConfig setWhiteListAddresses(String commaSeparatedList)
```
Comment (electrum) [inline presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:362]: We need a setter here
```java
@Config("cassandra.load-policy.allowed-addresses")
public CassandraClientConfig setAllowedAddresses(String allowedAddresses)
{
    this.allowedAddresses = SPLITTER.splitToList(allowedAddresses);
    return this;
}

File: presto-docs/src/main/sphinx/connector/cassandra.rst (hunk 39 above)
Comment (bentito) [reply presto-docs/src/main/sphinx/connector/cassandra.rst:140]: ✅

File: presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java (hunk 40 above)
Comment (bentito) [reply presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java:61]: ✅

File: presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java (hunk 41 above)
Comment (bentito) [reply presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraClientConfig.java:103]: ✅

File: presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java (hunk 42 above)
Comment (bentito) [reply presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraClientConfig.java:362]: ✅

**Comment by electrum** [comment]
Thanks!

**Pull request merged**